src/Classes/Particles.ts

```diff
@@ -25,6 +27,7 @@ export class Particles {
         this.container = container;
         this.array = [];
         this.interactionsEnabled = false;
+        this.spatialGrid = new SpatialGrid(this.container.canvas.dimension, 20);
```

Comment:
Why 20? Use Constants class for it, so it's readable

Reply:
Moved back into the spatial grid class as preset param.